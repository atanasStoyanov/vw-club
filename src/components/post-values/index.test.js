import React from 'react';
import renderer from 'react-test-renderer';
import PostValues from '.';

describe('PostValues Component', () => {
    it('should render post values with comment=1, likes=2, author="Pesho:', () => {
        const comments = ['test'];
        const likes = ['test', 'test'];
        const author = {
            username: 'Pesho'
        };

        const tree = renderer.create(
            <PostValues comments={comments} likes={likes} author={author} />)
        expect(tree).toMatchSnapshot();
    })
})