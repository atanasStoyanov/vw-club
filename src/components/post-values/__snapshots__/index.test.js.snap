// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostValues Component should render post values with comment=1, likes=2, author="Pesho: 1`] = `
<div>
  <p>
    <span>
      <small>
        Comments
        :
      </small>
       
      1
    </span>
  </p>
  <p>
    <span>
      <small>
        Likes
        :
      </small>
       
      2
    </span>
  </p>
  <p>
    <span>
      <small>
        Author
        :
      </small>
       
      Pesho
    </span>
  </p>
</div>
`;
